src/settings-ui/Settings.UI/SettingsXAML/Views/PowerLauncherPage.xaml

```diff
@@ -13,6 +14,293 @@
     AutomationProperties.LandmarkType="Main"
     mc:Ignorable="d">
 
+    <Page.Resources>
+        <Style x:Key="OptionSeparator" TargetType="Rectangle">
+            <Setter Property="Height" Value="1" />
+            <Setter Property="HorizontalAlignment" Value="Stretch" />
+            <Setter Property="Fill" Value="{ThemeResource CardStrokeColorDefaultBrush}" />
+        </Style>
+
+        <converters:RunOptionTemplateSelector
+            x:Key="RunOptionTemplateSelector"
+            CheckBoxComboBoxTemplate="{StaticResource CheckBoxComboBoxTemplate}"
+            CheckBoxMultilineTextBoxTemplate="{StaticResource CheckBoxMultilineTextBoxTemplate}"
+            CheckBoxNumberBoxTemplate="{StaticResource CheckBoxNumberBoxTemplate}"
+            CheckBoxTemplate="{StaticResource CheckBoxTemplate}"
+            CheckBoxTextBoxTemplate="{StaticResource CheckBoxTextBoxTemplate}"
+            ComboBoxTemplate="{StaticResource ComboBoxTemplate}"
+            EmptyTemplate="{StaticResource EmptyTemplate}"
+            MultilineTextBoxTemplate="{StaticResource MultilineTextBoxTemplate}"
+            NumberBoxTemplate="{StaticResource NumberBoxTemplate}"
+            TextboxTemplate="{StaticResource TextBoxTemplate}" />
+
+        <DataTemplate x:Key="CheckBoxTemplate" x:DataType="ViewModels:PluginAdditionalOptionViewModel">
+            <StackPanel HorizontalAlignment="Stretch" Orientation="Vertical">
+                <tkcontrols:SettingsCard
+                    MinHeight="0"
+                    Margin="1"
+                    Padding="0,6,0,12"
+                    Background="Transparent"
+                    BorderThickness="0,0,0,0"
+                    ContentAlignment="Left"
+                    CornerRadius="0">
+                    <controls:CheckBoxWithDescriptionControl
+                        Margin="56,0,0,0"
+                        Description="{x:Bind Path=DisplayDescription}"
+                        Header="{x:Bind Path=DisplayLabel}"
```

Comment:
Nit: `Path=` is optional :)

Reply:
`Path=` is widely used in the page (and across settings app). I prefer to keep the PR clear to review.
Feel free to open an issue/PR to polish it!